### PR TITLE
proto: add go_package option

### PR DESCRIPTION
**Pull request by zchee**
Add `go_package` proto option.

### CODE REVIEW

File: rootlesscontainers.proto [hunk 1]
```diff
@@ -19,6 +19,8 @@ syntax = "proto3";
 // If you want to extend the resource definition, please open a PR.
 package rootlesscontainers;
 
+option go_package = "github.com/rootless-containers/proto/go-proto";
```
Comment (AkihiroSuda) [inline rootlesscontainers.proto:22]: Could you update the go file too?
Then LGTM

Comment (zchee) [reply rootlesscontainers.proto:22]: 👍

**Comment by zchee** [comment]
@AkihiroSuda PTAL.

**Review by AkihiroSuda — APPROVED** [review]
Thanks, but next time please consider squashing commits

**Pull request merged**